### ISSUE TITLE
Proguard Error

### ISSUE THREAD

**Issue by ragdroid** [Mauin/RxFingerprint#27]
I am using version `1.2.2` and getting the following error.

```
Warning: com.mtramin.rxfingerprint.FingerprintObservable$$Lambda$1: can't find referenced class java.lang.invoke.LambdaForm$Hidden

```

Works fine on adding the following rule to proguard : 

```
#rxFingerprint
-dontwarn com.mtramin.rxfingerprint.**
```

**Comment by Mauin** [comment]
Thanks for the report @ragdroid. I'll investigate this one!

**Comment by Mauin** [comment]
This will go out with the next release. I'll first release the 2.x variant and then work on backporting all the changes to 1.x

**Comment by Mauin** [comment]
I just released v2.0.2 which should fix this issue. I'll start working on backporting the changes to the `1.x` branch but I can't give an ETA on the next `1.x` release.

**Comment by Mauin** [comment]
Just released v1.2.3 which should not need that additional ProGuard rule anymore. Let me know if this causes any more issues!
